modules/tinymce/src/plugins/codesample/test/ts/webdriver/CodeSampleCopyAndPasteTest.ts

```diff
@@ -50,51 +50,34 @@ describe('webdriver.tinymce.plugins.codesample.CodeSampleCopyAndPasteTest', () =
 
   it('TINY-8861: press enter after pasting a code sample should not add a newline inside the code', async () => {
     const editor = hook.editor();
-
+    editor.setContent('<p><br /></p><p><br /></p>');
     await TestUtils.pOpenDialogAndAssertInitial(hook.editor(), 'markup', '');
     TestUtils.setTextareaContent('test content');
     await TestUtils.pSubmitDialog(editor);
     TinyAssertions.assertSelection(editor, [], 0, [], 1);
     TinyAssertions.assertContentPresence(editor, { 'pre[data-mce-selected]': 1 });
 
     await pClickEditMenu(editor, 'Copy');
-    pressEnter(editor);
-    TinyAssertions.assertCursor(editor, [ 1 ], 0);
+    TinySelections.setCursor(editor, [ 1 ], 0);
     TinyAssertions.assertContentPresence(editor, { 'pre[data-mce-selected]': 0 });
 
     await pPaste(editor);
     TinyAssertions.assertSelection(editor, [], 1, [], 2);
     TinyAssertions.assertContentPresence(editor, { 'pre[data-mce-selected]': 1 });
 
+    // Pressing <enter> should do nothing.
```

Comment:
I wonder if this is really what we want to test here... I'm wondering if instead this should be pressing left or right to move beside and then press enter 🤷

Reply:
Yeah, I'm not sure. I was unsure how to preserve the intent of this test, as its underlying assumptions have changed.